WordPress/src/main/res/layout/site_picker_card.xml

```diff
@@ -1,66 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
This layout should be renamed to `site_picker_listitem` since it's no longer a card.